### ISSUE TITLE
Multiple queries on the same line

### ISSUE THREAD

**Issue by macobo** [amjith/pgcli#93]
Only the last query result is output.

Got:
```
$ pgcli -U postgres -h localhost _test_db
_test_db> select 1; select 2
+------------+
|   ?column? |
|------------|
|          2 |
+------------+
SELECT 1
```

Expected: PSQL
```
$ psql -U postgres -h localhost _test_db
psql (9.3.5)
Type "help" for help.

_test_db=# select 1; select 2;
 ?column? 
----------
        1
(1 row)

 ?column? 
----------
        2
(1 row)
```

**Comment by amjith** [comment]
This is a tricky one (or maybe not). 

We'll have to split by semi-colons and run them one at a time using pyscopg2 and print the results.

But the splitting has to take into account that a semi-colon could be inside a quoted string.

OT: I noticed in the pasted output, that the headers have ? around the column name. Is there a problem with displaying unicode chars?

**Comment by macobo** [comment]
I *think* sqlparse has a function for doing the splitting, I'll dig a bit and do a PR.

Re OT: as psql does the same thing I think it's the intended behavior.

**Comment by amjith** [comment]
This is now released in v0.12.0.

https://pypi.python.org/pypi/pgcli/0.12.0
